README.md

```diff
@@ -139,6 +140,26 @@ $ remme account transfer-tokens \
 }
 ```
 
+### Public key
+
+Get list of the public keys by account address — ``remme public-key get-list``:
```

Comment:
`Get list of the public keys ...` > `Get a list of the addresses of the public keys...`.

Apply the same to the code documentation.

Reply:
@dmytrostriletskyi, done.